alicloud/resource_alicloud_rkv_security_ips.go

```diff
@@ -0,0 +1,128 @@
+package alicloud
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/r-kvstore"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAlicloudRKVSecurityIPs() *schema.Resource {
```

Comment:
I think, at present, there is no need to add an extra resource 'resourceAlicloudRKVSecurityIPs' and just put security_ips into resource kvstore_instance is ok.

Reply:
If OK for you let's leave it in a dedicated resource provider for now. I think this is much more convenient than defining nested arrays in the parent resource. What do you think?

Reply:
If you can, I think SecurityIPs can be used to a new resource that can support rds and kvstore.